pkg/cmd/delete.go

```diff
@@ -66,11 +66,18 @@ func NewCompDeleteCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Comm
 		Annotations: map[string]string{
 			types.TagCommandType: types.TypeApp,
 		},
-		Example: "vela comp delete frontend",
+		Example: "vela comp delete frontend -a frontend",
 	}
 	cmd.SetOut(ioStreams.Out)
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		appName, err := cmd.Flags().GetString(App)
+		if err != nil {
+			return err
+		}
+		if appName == "" {
+			return errors.New("must specify name of application, please add flag -a")
+		}
```

Comment:
If the appName not specified, we should also possible delete component, for example use compName as AppName

Reply:
In this `delete`  appName  must specified, and the info of cmd is `Delete Component From Application`

```bash
$ bin/vela comp delete nginx
Deleting Component 'nginx' from Application ''
Error: delete appconfig err resource name may not be empty
```

Reply:
```go
appName, err := cmd.Flags().GetString(App)
if err != nil {
	return err
}
o.AppName = appName
```

the appname for flag `-a` , If the appName not specified, `err, message := o.DeleteComponent()` will throw err